### PR TITLE
Made Command that fetches the permanent invite url of Guild.

### PR DESCRIPTION
**Pull request by wqyeo**
Based on Issue #42 

The user can do `>cake invite` to make the bot fetch a permanent invite url for the server.

Additionally, if the user only wanted to fetch *any* invite url, the user can do<br>`>cake invite false`<br>And it will fetch *any* invite url from the guild.

The command will warn the user if no suitable invite exists in the guild.
Additionally, the command will also warn the user if the command is invoked in a non-server.

### CODE REVIEW

File: Cake.Core/Discord/Modules/CakeModule.cs [hunk 1]
```diff
@@ -19,6 +22,70 @@ public CakeModule(CakeService service)
...
+                    await ReplyAsync(GetErrorMessageToSend());
+                } else {
+                    var inviteToUse = invites.FirstOrDefault();
+                    await ReplyAsync(inviteToUse.Url);
+                }
+            }
+
+            string GetErrorMessageToSend() {
+                if (permanentInvite) {
+                    return $"There is no permanent invites in this server! {System.Environment.NewLine} Ask your owner to create one.";
```
Comment (CakeAndBanana) [inline Cake.Core/Discord/Modules/CakeModule.cs]: There is a space behind thje new line, the next line will be looking wierd in chat.

Could you please remove the space behind the new line.
Same goes for the other error message.

https://i.imgur.com/qCEFbzi.png

File: Cake.Core/Discord/Modules/CakeModule.cs [hunk 2]
```diff
@@ -19,6 +22,70 @@ public CakeModule(CakeService service)
...
+                } else {
+                    return $"There is no invites in this server! {System.Environment.NewLine} Ask your owner to create one.";
```
Comment (CakeAndBanana) [inline Cake.Core/Discord/Modules/CakeModule.cs]: Space behind newline.

**Review by CakeAndBanana — CHANGES_REQUESTED** [review]
Everything worked fine, there is only a small misstake at the error message where there is a space behind the new lines.

File: Cake.Core/Discord/Modules/CakeModule.cs (hunk 1 above)
Comment (wqyeo) [reply Cake.Core/Discord/Modules/CakeModule.cs]: Fixed, it.<br>Also, fixed newline indentation just to go with the supposed coding style in this document.

**Review by wqyeo — COMMENTED** [review]
Fixed it<br>Also, added newline indentation respectively to follow the coding style in this document.

**Review by CakeAndBanana — APPROVED** [review]
Thanks for the PR.

I never thought I would get a random PR out of nowhere so most issues are barebone or haven't got any information in them 😬.

If you have any questions about the other issues, feel free to ask in the discord.
https://discord.gg/bWMvT8P

Just ping me or Slicedd in general chat

**Pull request merged**